docs/topics/modules/associating_work_items_with_an_iteration.adoc

```diff
@@ -3,12 +3,14 @@
 
 To associate a Work Item with an Iteration:
 
-. Click the three dots adjoining the Work Item and click btn:[Associate with Iteration] from the displayed options. The `Associate with Iteration` dialog box appears and displays which Iteration, if any, the Work Item is associated with.
+. Click btn:[Plan] to see the Planner for your Space.
```

Comment:
I'm having a hard time finding the Plan button..I am on the Plan tab and I added a work item. Where should I see the Plan button. Thanks!